packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -81,12 +132,14 @@ const ProductFullDetail = props => {
                 <section className={classes.imageCarousel}>
                     <Carousel images={mediaGalleryEntries} />
                 </section>
+                {formErrorSection}
                 <section className={classes.options}>{options}</section>
                 <section className={classes.quantity}>
                     <h2 className={classes.quantityTitle}>Quantity</h2>
                     <Quantity
                         initialValue={quantity}
                         onValueChange={handleSetQuantity}
+                        message={errors.get('quantity') || null}
```

Comment:
The `null` may not be needed here. If there are no quantity errors this'll be `undefined`. Doesn't look like `Quantity` differentiates between them.

Reply:
Addressed in 8dcd4f9